jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -919,145 +919,21 @@ ecma_number_to_int32 (ecma_number_t num) /**< ecma-number */
   *
   * Note:
   *      The calculated values correspond to s, n, k parameters in ECMA-262 v5, 9.8.1, item 5:
-  *         - s represents digits of the number;
-  *         - k is the number of digits;
-  *         - n is the decimal exponent.
+  *         - parameter out_digits_p corresponds to s, the digits of the number;
+  *         - parameter out_decimal_exp_p corresponds to n, the decimal exponent;
+  *         - return value corresponds to k, the number of digits.
   */
-void
+lit_utf8_size_t
 ecma_number_to_decimal (ecma_number_t num, /**< ecma-number */
-                        uint64_t *out_digits_p, /**< [out] digits */
-                        int32_t *out_digits_num_p, /**< [out] number of digits */
+                        lit_utf8_byte_t *out_digits_p, /**< buffer to fill with digits */
```

Comment:
Missing [out] tag for output parameter.
`/**< [out] buffer to fill with digits */`